pkg/replication/replication.go

```diff
@@ -315,14 +313,16 @@ func (r *replication) WaitForReplication() {
 // Listen for errors in lock renewal. If the lock can't be renewed, we need to
```

Comment:
This function still does two things… maybe this comment should call that out.

Of course in an ideal world all functions would do precisely one thing, but that may not be possible given the way the channels flow around here.